doc/website/blog/2024-07-09-v025-row-level-security.mdx

```diff
@@ -0,0 +1,161 @@
+indexes, and triggers. This allows you to version control your policies, track changes, and apply them consistently
+across your environments.
+
+To get started with RLS in Atlas, first upgrade to the most recent version.
+
+<InstallationInstructions/>
+
+RLS is available to Atlas Pro users only. Get your free Atlas Pro account today by running:
+
+```shell
+atlas login
+```
+
+Next, you can define your RLS policies in your Atlas schema file (schema.hcl) using the new `policy` block:
+
+```hcl
+policy "employee_policy" {
+  on    = table.employees
+  for   = SELECT
+  to    = [PUBLIC]
+  using = "(current_user = employee_role)"
+}
+```
+
+This HCL snippet defines an RLS policy named `employee_policy` on the `employees` table, allowing only users whose
+`employee_role` matches their PostgreSQL username to `SELECT` rows from the table.
+
+Next, you need to enable RLS on the table:
+
+```hcl
+table "employees" {
+  schema = schema.public
+  column "employee_role" {
+    type = text
+  }
+  row_security {
+    enabled  = true // ENABLE ROW LEVEL SECURITY
+  }
+}
+```
+
+Finally, run `atlas schema apply` to apply the changes to your database!
+
+To learn more about RLS using Atlas, check out our [documentation](/atlas-schema/hcl#row-level-security-policies).
+
+### Introducing Atlas Pro
+
+Since launching Atlas Cloud a little over a year ago, we have been working hard with our users and customers to make
+Atlas as easy and simple to use as possible.
+
+One point of confusion we have encountered, especially around our pricing plans, was how users who currently don't
+want (or can't) use Atlas Cloud for their CI/CD pipelines can get access to the advanced CLI features that Atlas offers.
+Previously, teams needed to buy Cloud quota to get access to the CLI, which didn't make a lot of sense.
+
+To address some of these issues we are making some small changes to our pricing plans:
+
+Atlas now comes in three tiers:
+
+* **Open** - Our CLI, doesn't require creating an account and comes with a solid set of features (this is more than
+  enough for many of our users).
+* **Pro (previously "Business")** - An enhanced version of our CLI, which includes support for advanced database
+  features and drivers.
+
+  It will cost $9/month/user, but users get their first 3 seats per company for free when they sign up. Pro users
+  also have access to Atlas Cloud (pricing remains the same).
+* **Enterprise** - our enterprise tier, targeted mostly at larger organizations or teams in regulated industries with
+  stricter compliance requirements.
+
+To learn more about our new plans, head over to our [updated pricing page](/pricing).
+
+### Wrapping Up
+
+That's all for this release! We hope you try out (and enjoy) all of these new features and find them useful.
+As always, we would love to hear your feedback and suggestions on our [Discord server](https://discord.com/invite/zZ6sWVg6NT).
+
+
+
+
```

Comment:
```suggestion
* **Pro (previously "Business")** - An enhanced version of our CLI, which includes support for advanced database
  features and drivers.
  It will cost $9/month/user, but users get their first 3 seats per company for free when they sign up. Pro users
  also have access to Atlas Cloud (pricing remains the same).
* **Enterprise** - our enterprise tier, targeted mostly at larger organizations or teams in regulated industries with
  stricter compliance requirements.

To learn more about our new plans, head over to our [updated pricing page](/pricing).

### Wrapping Up

That's all for this release! We hope you try out (and enjoy) all of these new features and find them useful.
As always, we would love to hear your feedback and suggestions on our [Discord server](https://discord.com/invite/zZ6sWVg6NT).
```